packages/react-tree-grid/stories/Default.stories.tsx

```diff
@@ -17,111 +16,117 @@ import {
 export const Default = () => {
   return (
     <TreeGrid aria-label="All meetings">
-      <TreeGridRowGroup>
-        <TreeGridRow groupOwner>
-          <TreeGridCell header>Today</TreeGridCell>
-          <TreeGridCell aria-colspan={3}>
-            <Button>Header action</Button>
-          </TreeGridCell>
-        </TreeGridRow>
-        <TreeGridRow>
-          <TreeGridCell header>
-            Monthly townhall, 10:00 AM to 11:00 AM
-          </TreeGridCell>
-          <TreeGridCell>
-            <Button>Chat with participants</Button>
-          </TreeGridCell>
-          <TreeGridCell>
-            <Menu>
-              <MenuTrigger disableButtonEnhancement>
-                <Button>Toggle menu</Button>
-              </MenuTrigger>
+      <TreeGridRow
+        subtree={
+          <>
+            <TreeGridRow>
```

Comment:
Not sure if you're going to do a documentation pass later too, but would be nice to make these subtrees separate components

Reply:
maybe have an example too with manual `aria-level` set? It seems that for the case we are designing this for, it should be feasible to abstract theses to domain specific examples

```ts
const Header = <TreeGridRow aria-level="1" />

const Content = <TreeGridRow aria-level="2" />

<Header />
<Content />
<Content />
<Content />
<Content />
<Header />
<Content />
<Content />
```

Reply:
We can definitely have an example without using `subtree` and letting the user manually control depth, but unfortunately this is more than just setting `aria-level`. It's also about controlling visibility, so this example that you just gave would be a little bit more complex than just that 😅